tests/ci/docker_compose.sh

```diff
@@ -7,6 +7,14 @@ docker pull $DOCKER_IMAGE
 
 docker-compose up -d
 
-sleep 120
+#check if nuoadmin is up
 
-docker exec "$(docker container ls -f name=nuoadmin -q)" nuocmd check database --db-name hockey --check-running --num-processes 2 --timeout 300
+while [ "$(docker container inspect -f '{{.State.Running}}' "$(docker ps -f name=nuoadmin -q)")" = "false" ]
```

Comment:
I think you should first check that `docker ps -f name=nuoadmin -q` actually returns anything, and then check its state. It is possible that the container has not actually been started.